src/Tempest/Console/Scheduler/NullShellExecutor.php

```diff
@@ -6,7 +6,7 @@
 
 use Tempest\Console\ShellExecutor;
 
-class NullShellExecutor implements ShellExecutor
+final class NullShellExecutor implements ShellExecutor
```

Comment:
@brendt - We can't do this quite yet. Mocks were used for the testing of this class, which then fails tests because it is final.

Reply:
We shouldn't use mocks then :p Keep it in, I'll fix it

Reply:
@brendt - Totally agreed that we shouldn't be using that approach. Was going to leave in for this PR and create an issue/fix in another PR.